drizzle-zod/src/mysql/index.ts

```diff
@@ -0,0 +1,243 @@
+import { GetColumnConfig } from 'drizzle-orm';
+import {
+	AnyPgColumn,
+	AnyPgTable,
+	GetTableConfig,
+	PgBigInt53,
+	PgBigInt64,
+	PgBigSerial53,
+	PgBigSerial64,
+	PgBoolean,
+	PgDate,
+	PgDoublePrecision,
+	PgEnumColumn,
+	PgEnumColumnConfig,
+	PgInteger,
+	PgInterval,
+	PgNumeric,
+	PgReal,
+	PgSerial,
+	PgSmallInt,
+	PgSmallSerial,
+	PgText,
+	PgTextConfig,
+	PgTime,
+	PgTimestamp,
+	PgUUID,
+	PgVarchar,
+} from 'drizzle-orm/mysql-core';
+import { getTableColumns } from 'drizzle-orm/pg-core/utils';
+import { Simplify } from 'drizzle-orm/utils';
+import { z } from 'zod';
+
+type SnakeToCamelCase<S extends string> = S extends `${infer T}${'_' | '-'}${infer U}`
+	? `${T}${Capitalize<SnakeToCamelCase<U>>}`
+	: S;
+
+type ToSnakeCase<S extends string> = S extends `${infer T}${'' | '-'}${infer U}`
+	? `${T extends Capitalize<T> ? Lowercase<T> extends Capitalize<T> ? '' : '_' : ''}${Lowercase<T>}${ToSnakeCase<U>}`
+	: S;
+
+function toCamelCase(value: string): string {
+	return value.replace(/([-_][a-z])/gi, ($1) => $1.toUpperCase().replace('-', '').replace('_', ''));
+}
+
+function toSnakeCase(value: string): string {
+	return value.replace(/([A-Z])/g, ($1) => `_${$1.toLowerCase()}`);
+}
+
+type MaybeOptional<TColumn extends AnyPgColumn, TType extends z.ZodTypeAny, TNoOptional extends boolean = false> =
```

Comment:
`AnyPgColumn` is used for postgres specific functions. We need to use `AnyMySqlColumn` here